FreeRTOS-Plus/Demo/MQTT_Mutual_Auth_Qemu_mps2/mbedtls_freertos_port.c

```diff
@@ -0,0 +1,259 @@
+/*
```

Comment:
Is this different from FreeRTOS-Plus/Source/Utilities/mbedtls_freertos/mbedtls_freertos_port.c?

Reply:
Yes, mbed_tls_platform_entropy_poll used windows system calls and it was updated for qemu